MockHttpServer.UnitTests/MockServerTests.cs

```diff
@@ -306,18 +306,19 @@ public void TestSpecifiedMethods()
             }
         }
 
-        [TestMethod]
-        public void TestSpecifiedHostName()
-        {
-            var client = CreateRestClient();
-            string expectedResult = "Result";
-
-            using (new MockServer(TestPort, "", (req, rsp, prm) => expectedResult, "+"))
-            {
-                var result = client.Execute(new RestRequest("", Method.POST));
-                Assert.AreEqual(expectedResult, result.Content);
-            }
-        }
+        // TODO: This test didn't work to begin with
```

Comment:
Any details on what the error you were seeing was?  The test was definitely passing before, as you can see from the most recent Appveyor logs.

Reply:
Interestingly when I hooked up AppVeyor the test on master passes, but when I ran locally it fails. The error was as follows:

```
MockHttpServer.UnitTests.MockServerTests.TestSpecifiedHostName threw exception: 
System.Net.HttpListenerException: Access is denied
```

Reply:
@dansiegel Try running VS as an administrator.

Reply:
well crap that actually does work... I'll back that change out.